scripts/check-theme-lark-imports.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+'use strict';
+
+/* eslint-env node */
+
+const chalk = require( 'chalk' );
+const path = require( 'path' );
+const fs = require( 'fs' );
+const glob = require( 'glob' );
+
+const THEME_LARK_DIR_PATH = path.resolve( __dirname, '..', 'packages', 'ckeditor5-theme-lark', 'theme' );
+const REGEX_FOR_INDEX_IMPORTS = /(?<=@import ".\/)(.*)(?=";)/gm;
```

Comment:
I made this regexp more flexible a bit so that all import paths were taken. 

But by design in `index.css` there should be only paths that are on the same level or being deeper in directory structure.